bluebutton-data-pipeline-rif-extract/src/main/java/gov/hhs/cms/bluebutton/datapipeline/rif/extract/s3/DataSetManifest.java

```diff
@@ -0,0 +1,138 @@
+package gov.hhs.cms.bluebutton.datapipeline.rif.extract.s3;
+
+import java.time.Instant;
+import java.util.Arrays;
+import java.util.List;
+
+import javax.xml.bind.annotation.XmlAccessType;
+import javax.xml.bind.annotation.XmlAccessorType;
+import javax.xml.bind.annotation.XmlAttribute;
+import javax.xml.bind.annotation.XmlElement;
+import javax.xml.bind.annotation.XmlRootElement;
+import javax.xml.bind.annotation.adapters.XmlJavaTypeAdapter;
+
+import com.migesok.jaxb.adapter.javatime.InstantXmlAdapter;
+
+import gov.hhs.cms.bluebutton.datapipeline.rif.model.RifFileType;
+
+/**
+ * Represents the <code>manifest.xml</code> files that detail which specific
+ * files are included in a transfer from the CMS Chronic Conditions Warehouse to
+ * the Blue Button API backend.
+ */
+@XmlRootElement
+@XmlAccessorType(XmlAccessType.FIELD)
+public final class DataSetManifest {
```

Comment:
Spent too much time fighting with JAXB this morning trying to implement it, so deferring it into http://issues.hhsdevcloud.us/browse/CBBD-113.